fenix/app/metrics.yaml

```diff
@@ -376,6 +376,24 @@ events:
     metadata:
       tags:
         - PrivateBrowsing
+  opened_ext_pdf:
+    type: event
+    description: |
+      A user opened a PDF with Fenix from another app
+    extra_keys:
+      referrer_is_fenix:
+        description: |
+          If the PDF was opened from Fenix itself (for example from the Download notification)
+        type: boolean
```

Comment:
Should we also send an event when users click a PDF file on the "Downloads" screen?  :)

Reply:
@Amejia481 maybe but I don't think it should be a blocker for this PR IMHO. It is less important

Reply:
Actually with this patch we send "true" for both cases: opening from the Downloads screen and opening from the Donwload notification (in app, or in the Notification Tray). So I guess this is already covered

Reply:
Perfect, from my side it looks good we just need the data steward approval too :)

Reply:
@titooan I'm not sure but product maybe interested into identify how many PDF viewer requests are coming from our app or other apps.